packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -297,17 +297,12 @@ const IndicatorComponent = (
   );
 
   const numUserNotMentioned = typingUsers.length - typingUsersMentioned.length;
-  const spanElements: JSX.Element[] = getSpanElements(
-    strings,
-    usersElement,
-    typingUsersMentioned.length,
-    numUserNotMentioned
-  );
+  const spanElements: JSX.Element[] = getSpanElements(strings, usersElement, typingUsers.length, numUserNotMentioned);
 
   const labelString = getIndicatorString(
     strings,
     getNamesString(typingUsersMentioned, strings.delimiter),
-    typingUsersMentioned.length,
+    typingUsers.length,
```

Comment:
It makes a lot of sense to use typingUsers.length to set typing user count.
Just a little bit curious about what scenarios will current code cause problems? Inside of getNameString logic, it only depends on whether typingUserCounts > 1 or >0, then it looks the same to me for typingUsers.length and typingUserMentioned.length (But still we should fix this logic)

Reply:
@PorterNan 
because we passed typingUsersMentioned as the 3rd param of getSpanElements (which is numTypingUsers in line 194), we ended up always having numTypingUsers at 1 in case of numTypingUsersAbbreviated > 1 , meaning always having 'UserName is typing ...' when we should have 'UserName and 1 other are typing ...' or 'UserName and x others are typing ...'

Same issue with getNamesString as it was a copy/past of params :(